package.json

```diff
@@ -53,7 +53,7 @@
     "@electron-toolkit/tsconfig": "^1.0.1",
     "@types/node": "^18.19.5",
     "electron-builder": "^24.9.1",
-    "electron-vite": "https://github.com/matthme/electron-vite.git#forward-cli-args-dist",
```

Comment:
This was a workaround that then got merged into the official package and is not needed anymore (and I think actually never was needed for hc-spin but the package.json had been copied from the launcher repo initially)